server/index.js

```diff
@@ -187,38 +189,36 @@ Visit the documentation for more information: https://bpanel.org/docs/configurat
     return clientsMap;
   }, new Map());
 
-  // Init bsock socket server
-  const socketHttpServer = http.createServer();
-  bsock.attach(socketHttpServer);
-
   // Init app express server
   const app = express.Router();
   const port = process.env.PORT || 5000;
   const bsockPort = process.env.BSOCK_PORT || 8000;
   app.use(bodyParser.json());
   app.use(cors());
 
+  // create new SocketManager
+  // TODO: consolidate to single logger using blgr and remove other dependency
```

Comment:
Maybe in the next logger related PR there could be something around handling the log level instead of hardcoding it to `'info'`?

Reply:
Yeah, I think that's a good idea. That could be passed in through some cli or env argument.

Reply:
added to backlog